packages/spectrum/test/renderers/SpectrumAnyOfRenderer.test.tsx

```diff
@@ -144,29 +122,32 @@ describe('Spectrum anyOf renderer', () => {
         },
       },
     };
+
     const uischema: ControlElement = {
       type: 'Control',
       scope: '#/properties/myThingsAndOrYourThings',
     };
 
-    wrapper = mountForm(uischema, schema, {});
+    const { getByRole, getAllByRole } = renderForm(uischema, schema, {});
+    await waitForAsync(); // TODO: how to do it with the testing-library...
 
-    await waitForAsync();
-
-    wrapper.update();
+    let tablist = getByRole('tablist');
+    let tabs = within(tablist).getAllByRole('tab');
+    let firstItem = tabs[0];
+    expect(firstItem).toHaveAttribute('aria-selected', 'true');
+    triggerPress(firstItem);
 
-    selectanyOfTab(wrapper, 1);
-    const nrOfRowsBeforeAdd = wrapper.find('tr');
-    clickAddButton(wrapper, 2);
-    const nrOfRowsAfterAdd = wrapper.find('tr');
```

Comment:
It's not the tab switching that is adding the mything, it's the click on the add button (or the '+'-Button, see `clickAddButton`). I suppose there should initially be one element present due to the given data/state.

Reply:
ah - this got lost somehow

Reply:
fixed it